nsqd/guid/guid.go

```diff
@@ -1,4 +1,4 @@
-package nsqd
+package guid
```

Comment:
this whole package would be going away in #625, so I wouldn't spin your wheels on it